test/collection.js

```diff
@@ -1437,6 +1437,17 @@ QUnit.test("#1894 - `sort` can optionally be turned off", function(assert) {
     assert.ok(collection.comparator === comparator);
   });
 
+  QUnit.test("Pass falsey for `models` for empty Col with `options`", function(assert) {
+    assert.expect(6);
+    _.forEach([undefined, null, false], function(falsey) {
+      var MyModel = Backbone.Model.extend();
+      var collection = new Backbone.Collection(falsey, {model: MyModel});
+
+      assert.ok(collection.model === MyModel);
```

Comment:
Good idea. I've updated the code to use your example with one small change: I had to add the `models` argument to the `initialize()` method. Since I had the value, I figured I'd test it as well. Let me know if that feels excessive.